source/python/run_api.py

```diff
@@ -49,9 +49,9 @@ def article(doc_id):
     session = Session()
     analysis = session.query(Analysis) \
         .filter(Analysis.document_id == doc_id).one()
-    coords = [l.latlong.split(",")[::-1]
+    coords = [tuple(l.latlong.split(","))
```

Comment:
Instead of building a list and then calling `set` on it, you can make the set directly:
```
coords = {tuple(l.latlong.split(",") for f in analysis.facts for l in f.locations}
```